src/test/java/uk/gov/ch/developer/docs/controller/AbstractPageControllerTest.java

```diff
@@ -14,6 +9,9 @@
 import uk.gov.ch.developer.docs.session.SessionService;
 import uk.gov.companieshouse.session.Session;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
```

Comment:
IntelliJ doing strange things again

Reply:
Try using the reformat code tool to see if it solves this.

Reply:
Try using the CH-Style configuration in your development environment.

see: Configs to assist with complying with Java Coding Standards in https://companieshouse.atlassian.net/wiki/spaces/TCE/pages/1097924715/Win+10+VM+for+Development for where to get it and how to install it.

Reply:
Done